src/trigger-method.js

```diff
@@ -33,11 +33,7 @@ Marionette._triggerMethod = (function() {
 
     // trigger the event, if a trigger method exists
     if (_.isFunction(context.trigger)) {
-      if (noEventArg + args.length > 1) {
-        context.trigger.apply(context, noEventArg ? args : [event].concat(_.drop(args, 0)));
-      } else {
-        context.trigger(event);
-      }
+      context.trigger.apply(context, noEventArg ? args : [event].concat(_.drop(args, 0)));
```

Comment:
Hmm I don't actually know.  I didn't change this line at all.  
From here: https://github.com/marionettejs/backbone.marionette/commit/1316716980080173eccb18f47dfa5c80485645fb#diff-a20005534e3338e0b6f5c03de7ba9952R38
But the `,0` can be dropped (no pun intended) with no consequence with lodash right?